CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # Changelog
 
+### Version 1.3.0
+    
+- BREAKING CHANGES
+    - Tealium takes now a TealiumConfig object with different properties at initialisation. 
```

Comment:
Not need to detail the changes on Tealium side.
To keep a clear changelog, you can just mention the support of Tealium's version in our library which changed to v2 from v1 should be enough.